app/helpers/mods_display/record_helper.rb

```diff
@@ -17,6 +17,14 @@ def mods_record_field(field, delimiter = nil, component: ModsDisplay::FieldCompo
       render component.new(field: field, delimiter: delimiter, value_transformer: block)
     end
 
+    def mods_record_definition_field(field, delimiter: nil, label_html_attributes: {}, &block)
+      render ModsDisplay::FieldComponent.new(field: field, delimiter: delimiter, label_html_attributes: label_html_attributes, value_transformer: block)
+    end
+
+    def mods_record_row_field(field, delimiter: ', ', &block)
```

Comment:
I agree, but this is consistent with the existing helpers. It would be confusing not to have helper.

That being said, a follow-up refactor to clean up the helper mess in mods_display is probably a good idea.